src/tmx/TmxUtils/src/SNMPClient.cpp

```diff
@@ -0,0 +1,323 @@
+#include "SNMPClient.h"
+
+namespace tmx::utils {
+    
+    // Client defaults to SNMPv3
+    snmp_client::snmp_client(const std::string& ip, const int& port, const std::string& community, 
+        const std::string &snmp_user, const std::string &securityLevel, const std::string &authPassPhrase, int snmp_version, int timeout)
+
+        : ip_(ip), port_(port), community_(community),snmp_version_(snmp_version), timeout_(timeout)
+    {
+        
+        PLOG(logDEBUG1) << "Starting SNMP Client";
+        PLOG(logDEBUG1) << "Target device IP address: " << ip_;
+        PLOG(logINFO) << "Target device NTCIP port: " << port_;
```

Comment:
This logging does not may sense to me. NTCIP port is probably incorrect here. Also we probably want all these log statements to share a log level.

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/564/commits/c0bac101e0cef01c154929024be6ab01b263785b